docs/Installation.md

```diff
@@ -31,42 +31,54 @@ This instruction covers the following operating systems:
 
 ### 1. CentOS
 
-* Install Httpd (Apache), if it is not installed:
+#### Install Apache
 
 ```sh
 sudo yum update
 sudo yum install httpd
 ```
 
+
 #### Install Zonemaster Web GUI
 
 ```sh
 wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.2.1/zonemaster_web_gui.zip -O zonemaster_web_gui.zip
```

Comment:
Can you make the URL the last argument here ?

Reply:
I suppose. Why is that better?

Reply:
I would say that usually options come first on the command line, and this is how [the syntax is defined-(https://www.gnu.org/software/wget/manual/wget.html#Invoking). Moreover if you copy paste the command, you want to know where the file is stored more than where it comes from (but this is last argument might be completely subjective).

However, I see you updated the PR in a way that makes this remark irrelevant.